deploy/helm/values.yaml

```diff
@@ -312,13 +312,36 @@ admissionWebhooks:
 dataProtection:
   enabled: true
   enableVolumeSnapshot: false
-  backupPVCName: kb-backup-data
+  backupPVCName: ""
```

Comment:
Why remove this PVC name?

Reply:
by default, use backupRepo to replace PersistentVolumeClaim